src/react/components/pages/editorPage/editorPage.scss

```diff
@@ -251,5 +233,76 @@ canvas {
 }
 
 .asset-error {
+    font-size: 85%;
+    font-weight: 500;
     margin: auto;
 }
+
+.badge-visited, .badge-tagged {
+    color: #fff;
+    position: absolute;
+    z-index: 2;
+    right: 0.5em;
+    top: 0.5em;
+    text-shadow: 0px 0px 4px #000
+}
+
+.badge-tagged {
+    background-color: rgba(green, 0.9);
+    border: 1px solid $lighter-2;
+}
+
+.badge-visited {
+    background-color: rgba(darkorange, 0.9);
+    border: 1px solid $lighter-3;
```

Comment:
remove border for uniformity with other icons

Reply:
Same as above.